docs/work-with-reports/ViewLaunches.mdx

```diff
@@ -173,6 +173,23 @@ Also, you can use the logs sorting by time, and filtering logs to find a certain
 
 <MediaViewer src="https://youtu.be/_g0GW16sXGU" alt="Log Messages in our automated testing tool" type="video" />
 
+### Timestamps
+
+In ReportPortal version 24.2, microseconds have been added to timestamps to provide a more precise log sequence.
+Previously, some logs shared identical timestamps at the millisecond level, which led to issues with log ordering.
+
+With this update, you have the flexibility to view timestamps in two formats:
+
+1. milliseconds
+
+<MediaViewer src={require('./img/view-launches/Milliseconds.png')} alt="Date & time" />
+
+2. milliseconds with microseconds
+
+<MediaViewer src={require('./img/view-launches/Microseconds.png')} alt="Time with number of microseconds" />
+
+You can easily switch between these options with a simple click.
+
```

Comment:
Maybe also add a message that the choice will be saved to `localStorage` to save between sessions in a particular browser?